wandb/sdk/internal/internal_api.py

```diff
@@ -2216,6 +2216,92 @@ def upsert_run(
             server_messages,
         )
 
+    @normalize_exceptions
+    def rewind_run(
+        self,
+        run_name: str,
+        metric_name: str,
+        metric_value: float,
+        program_path: Optional[str] = None,
+        entity: Optional[str] = None,
+        project: Optional[str] = None,
+        num_retries: Optional[int] = None,
+    ) -> dict:
+        """Update a run.
```

Comment:
could you provide a better explantation for this function?

Reply:
Oof, sorry. This was AI generated ... fixed.